src/ProjectDataBase/Tables/TicketStatus.sql

```diff
@@ -0,0 +1,5 @@
+﻿CREATE TABLE [dbo].[TicketStatus]
+(
+    [Id] INT NOT NULL PRIMARY KEY IDENTITY,
+    [StatusName] NVARCHAR(50) NOT NULL
```

Comment:
Name can be used instead of StatusName. Usual convention about Lookup tables Id+Name fields + Additional fields if required